core/templates/pages/topics-and-skills-dashboard-page/topics-and-skills-dashboard-page.component.html

```diff
@@ -214,7 +214,7 @@
                   </oppia-filtered-choices-field>
                 </div>
                 <div *ngIf="activeTab === TAB_NAME_SKILLS">
-                  <oppia-filtered-choices-field [choices]="skillClassrooms"
+                  <oppia-filtered-choices-field [choices]="classrooms"
```

Comment:
So, the variable was named incorrectly? (As in, does `classrooms` already exist in the component?)

Reply:
Yeah there is classrooms in the component file but no skillClassrooms.